### DISCUSSION TITLE
Guess job

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#546] [Galaxy Trivia] [answered]
**LLRID**
**HOTOT**
**LINLIFG**
**WNCOR**

**Comment by peterbaccaro** [comment]
**DENTIST**

DRILL
TOOTH
FILLING
CROWN

**Comment by thuongtruong1009** [comment, marked as answer]
LLRID: Drill
HOTOT: Tooth
LINLIFG: Filling
WNCOR: Crown
